configurations/giz_pipeline.py

```diff
@@ -0,0 +1,316 @@
+from core_data_modules.cleaners import swahili
+
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="GIZ",
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/POOL-KENYA-REBUILD"
```

Comment:
Can you add a todo explaining why it's to the rebuild and that it should be corrected when possible?